docs/mllib-evaluation-metrics.md

```diff
@@ -462,10 +462,10 @@ $$rel_D(r) = \begin{cases}1 & \text{if $r \in D$}, \\ 0 & \text{otherwise}.\end{
       <td>Normalized Discounted Cumulative Gain</td>
       <td>
         $NDCG(k)=\frac{1}{M} \sum_{i=0}^{M-1} {\frac{1}{IDCG(D_i, k)}\sum_{j=0}^{n-1}
-          \frac{rel_{D_i}(R_i(j))}{\text{ln}(j+2)}} \\
+          \frac{rel_{D_i}(R_i(j))}{log_2(j+2)}} \\
```

Comment:
The base of the log doesn't matter as it will put a constant multiplier on every term in the ratio. Internally it's computed as `ln` as you note below. We could just write `log`, even, to indicate it doesn't matter.

Reply:
Yes. To make it consistent with the actual formula I wrote this. If we write ln or log, then numerator need to have 2^rel - 1.
But, this does not matter as long as rel is binary.